clients/ui/frontend/src/app/pages/modelRegistry/screens/RegisterModel/RegisterModelDetailsFormSection.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  FormGroup,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+  TextArea,
+  TextInput,
+} from '@patternfly/react-core';
+import React from 'react';
+import FormSection from '~/shared/components/pf-overrides/FormSection';
+import { UpdateObjectAtPropAndValue } from '~/shared/types';
+import { MR_CHARACTER_LIMIT } from './const';
+import { RegisterModelFormData } from './useRegisterModelData';
+
+type RegisterModelDetailsFormSectionProp<D extends RegisterModelFormData> = {
+  formData: D;
+  setData: UpdateObjectAtPropAndValue<D>;
+  hasModelNameError: boolean;
+  isModelNameDuplicate?: boolean;
+};
+const RegisterModelDetailsFormSection = <D extends RegisterModelFormData>({
+  formData,
+  setData,
+  hasModelNameError,
+  isModelNameDuplicate,
+}: RegisterModelDetailsFormSectionProp<D>): React.ReactNode => (
+  <FormSection
+    title="Model details"
+    description="Provide general details that apply to all versions of this model."
+  >
+    <FormGroup label="Model name" isRequired fieldId="model-name">
+      <TextInput
```

Comment:
Missing the field sets that @jenny-s51 added to the code before it was moved to this file.